images/build-drivers/build-drivers.sh

```diff
@@ -46,9 +53,11 @@ echo "******************************************************"
 start_docker
 
 cd driverkit/
-make -e TARGET_DISTRO="$1" specific_target
 
-test "${PUBLISH_S3}" == "true" && make publish_s3
+make $DBG_ENV_PARAMS specific_target
```

Comment:
Finally, seeing you Max using my beloved Makefiles makes me so happy! 🤗

Reply:
😄 Make is so powerful, why not using them!? Then, they're well-written I think!